openedx/features/subscriptions/api/v1/urls.py

```diff
@@ -0,0 +1,15 @@
+from django.conf.urls import include, url
+
+from openedx.features.subscriptions.api.v1.views import SubscriptionsListView, SubscriptionRetrieveUpdateView
+
+SUBSCRIPTION_URLS = ([
+    url(r'^$', SubscriptionsListView.as_view(), name='list'),
+    url(r'^(?P<subscription_id>.*)/$', SubscriptionRetrieveUpdateView.as_view(), name='retrieve_update'),
```

Comment:
nit: `name=subscriptions-retrieve-update`

Reply:
`SUBSCRIPTION_URLS` has `'subscriptions'` passed as namespace so when i do `include(SUBSCRIPTION_URLS)`, it should provide a syntax like `subscriptions:list` and `subscriptions:retrieve_update`. If u want me to change that to this I can do that as well.